mibidata/mibi_image.py

```diff
@@ -218,7 +216,7 @@ def __eq__(self, other):
         return False
 
     def __getitem__(self, channels):
-        """Equivalent to :meth:`~slice_data`."""
+        """Equivalent to :meth:`~slijce_data`."""
```

Comment:
???

Reply:
It was an accident... seems like I do not know what I am typing. Even with both hands.